### DISCUSSION TITLE
주문 조회 > 배송비 계산 문의

### DISCUSSION THREAD

**Discussion by circleplatform** [commerce-api-naver/commerce-solutions#65] [묻고 답하기] [answered]
안녕하세요. 서클플랫폼 이준원입니다.

커머스솔루션 API에서 제공 받은 주문 목록을 보여주다가, 배송비에 대해서 궁금한 점이 있어 연락드립니다.

먼저 저희는 `주문 조회` > `변경 상품 주문 내역 조회` / `상품 주문 상세 내역 조회` 이렇게 두 가지 API를 사용하고 있구요.

`상품 주문 상세 내역 조회` > `deliveryFeeAmount` 라는 컬럼으로 배송비를 보여주려고 하는데, 아래 케이스들을 저희가 나눠서 알 수 있을까요?

1. (합배송) 3개 상품의 배송비가 총 3,000원인 경우
2. (2/1 분리배송) 3개 상품의 배송비가 총 6,000원인 경우
3. (3 분리배송) 3개 상품의 배송비가 총 9,000원인 경우

혹시 추가적으로 제가 확인할 수 있는 부분이 있다면 언제든 알려주시면 제가 확인해보겠습니다.

감사합니다.

**Comment by commerce-api-naver** [comment, marked as answer]
안녕하세요. 커머스API 이민석입니다.

스마트스토어에서는 '배송비 묶음 그룹'이라는 개념이 있습니다.
동일 스토어에서 여러 상품을 주문하더라도 같은 '배송비 묶음 그룹'으로 지정된 상품 사이에서만 배송비 계산이 이루어집니다.
가령 '조건부 무료 배송' 조건이더라도 같은 배송비 묶음 그룹으로 지정된 상품의 주문 금액만 무료 배송 기준 금액으로 산정하여 배송비 면제/부과를 결정하는 식입니다.

이 정보는 '상품 주문 상세 내역 조회' API 내 `data[n].productOrder.packageNumber` 필드 값으로 제공합니다.
상품 정보 상 배송비 묶음 그룹은 개별 상품 단위로 설정할 수 있기에 주문건에서는 '상품주문번호(`productOrder` 필드)' 단위로 제공하며, 각 상품주문번호별로 `packageNumber` 필드 값이 동일하다면 같은 배송비 묶음 그룹에 속하여있기에 배송비가 1회 부과됩니다.

### 예시
| 상품주문번호 | `packageNumber` 필드 값 | `deliveryFeeAmount` 필드 값|
|---|---|---|
| 상품주문번호1 | 11111 | 3000 |
| 상품주문번호2 | 11111 | 3000 |
| 상품주문번호3 | 11111 | 3000 |
| 상품주문번호4 | 11112 | 2000 |

위 예시 주문건은 총 배송비가 5,000원(3000 + 2000) 부과되었습니다.

**Reply by circleplatform** [reply]
확인 감사합니다. 큰 도움이 되었습니다!
